plugin/trino-exchange/src/main/java/io/trino/plugin/exchange/FileSystemExchangeManagerFactory.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.exchange;
+
+import com.google.inject.Injector;
+import io.airlift.bootstrap.Bootstrap;
+import io.trino.spi.exchange.ExchangeManager;
+import io.trino.spi.exchange.ExchangeManagerFactory;
+import io.trino.spi.exchange.ExchangeManagerHandleResolver;
+import io.trino.spi.exchange.ExchangeSinkInstanceHandle;
+import io.trino.spi.exchange.ExchangeSourceHandle;
+
+import java.util.Map;
+
+import static java.util.Objects.requireNonNull;
+
+public class FileSystemExchangeManagerFactory
+        implements ExchangeManagerFactory
+{
+    @Override
+    public String getName()
+    {
+        return "distributed";
```

Comment:
Maybe just "filesystem" to match class naming; not sure though. It would be nice those to be consistent.